src/migratus/cli.clj

```diff
@@ -95,17 +98,109 @@
 
       :else (no-match-message args summary))))
 
+(defn util-date-to-local-datetime [util-date]
+  (if (= util-date nil)
+    nil
+    (let [instant (.toInstant util-date)
+          zone-id (ZoneId/systemDefault)
+          local-datetime (.atZone instant zone-id)]
+      local-datetime)))
+
+(defn formatted-date [date]
+  (if (nil? date)
+    nil
+    (core/format "%1tFT%<tTZ", date)))
+
+(defn parsed-migrations-data [cfg]
+  (let [all-migrations (migratus/all-migrations cfg)]
+    (map (fn [m] (let [{:keys [id name applied]} m
```

Comment:
try to avoid writing fn's inside map .
It makes the code harder to read. 

Well written code should read as prose (some say).

Define the function in let or using defn , with a name and use the name. 
The function should be named for what it does. 

If you can't find a name easy - it means the function is not that clear in what it does. 
Inline functions

Reply:
named the anonymous fn